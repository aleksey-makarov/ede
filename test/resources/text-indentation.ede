{
  "single": "lorem",
  "multi": "Navzdory všeobecnému\npřesvědčení\nLorem Ipsum není\nnáhodný text."
}
---
{{ single | justifyLeft(25) }}
{{ single | justifyRight(15) }}
{{ single | center(9) }}

{{ multi | indentLines(4) }}

{{ multi | prependLines("...") }}
