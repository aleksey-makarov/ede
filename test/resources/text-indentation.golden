lorem                    
          lorem
  lorem  

    Navzdory všeobecnému
    přesvědčení
    Lorem Ipsum není
    náhodný text.

...Navzdory všeobecnému
...přesvědčení
...Lorem Ipsum není
...náhodný text.
